extension-reference/mobile/build-your-own-extension/events/identity/events-handled-by-the-identity-service.md

```diff
@@ -1,39 +1,118 @@
-# Events Handled by the Identity Service
+# Events Handled by the Identity Extension
 
-## Hub Stared State
+## Identity Request Identity
 
-The shared state event is dispatched by the event hub after a shared state is created or updated. The event data contains the event owner and the name of the module to which the shared state belongs. This event is generated by the Event Hub.
+This event is used to retrieve data from the Identity extension in one of the following situations:
+
+- to retrieve the custom visitor identifiers
+- to retrieve the Experience Cloud ID (MID)
+- to append the visitor identifiers to an URL
+- to synchronize a visitor identifier or a list of identifiers with the Adobe Experience Cloud ID Service
+
+This event is generated by:
+
+- Calling the Identity `getIdentifiers` API
+- Calling the Identity `getExperienceCloudId` API
+- Calling the Identity `appendVisitorInfoForURL` (Android) / `appendToUrl` (iOS) API
+- Calling the Identity `syncIdentifier` or `syncIdentifiers` APIs
+
+### Event Details
+
+| **Event Type**               | **Event Source**                      | **Paired** | **Paired Event**                                             |
+| :--------------------------- | :------------------------------------ | :--------- | :----------------------------------------------------------- |
+| com.adobe.eventType.identity | com.adobe.eventSource.requestIdentity | Yes        | [Link To The Paired Event](../link/to/the/eventpage/andsection) |
 
 ### Data Payload Definition
 
 Here are the key-value pairs in this event:
 
-| **Key** | **Value Type** | **Optional** | **Description** |
-| :--- | :--- | :--- | :--- |
-| stateowner | String | N/A | N/A |
+| **Key**             | **Value Type**      | **Optional** | **Description**                                              |
+| :------------------ | :------------------ | :----------- | :----------------------------------------------------------- |
+| baseurl             | String              | Yes          | The base URL passed as parameter to the `appendVisitorInfoForURL` (Android) / `appendToUrl` (iOS) API |
+| visitoridentifiers  | Map<String, String> | Yes          | The visitor identifiers that need to be synced with the Experience Cloud ID Service. The keys of the map are the identifier types and the values are the associated identifiers |
+| authenticationstate | Integer             | Yes          | The VisitorID Authentication State represent (Unknown - 0, Authenticated - 1, Logged Out - 2) |
+| forcesync           | Boolean             | Yes          | Boolean indicating if this sync call should be sent to the Experience Cloud ID Service regardless of any changes to the identifiers |
+| issyncevent         | Boolean             | Yes          | Boolean indicating if this is an Identity sync event         |
```

Comment:
I will remove `forcesync` and `issyncevent` from the event data detailed description since these flags are not required anymore in the event data and Identity extension can detect when a sync event is required and force sync is required for boot event only.